integration/storage/extended_statefulset_with_storage_test.go

```diff
@@ -35,7 +36,9 @@ var _ = Describe("ExtendedStatefulSet", func() {
 		BeforeEach(func() {
 
 			essName := fmt.Sprintf("testess-%s", helper.RandString(5))
-			extendedStatefulSet = env.ExtendedStatefulSetWithPVC(essName, "pvc")
+			storageClass, ok := os.LookupEnv("OPERATOR_TEST_STORAGE_CLASS")
+			Expect(ok).To(Equal(true))
```

Comment:
It's just syntactic sugar, but there's also `BeTrue()`.

Reply:
Done.